identityClient/identityClient.go

```diff
@@ -150,6 +150,18 @@ func (c *E3dbIdentityClient) DescribeRealmRole(ctx context.Context, params Descr
 	return role, err
 }
 
+// UpdateRealmRole updates the specified realm role, returning error (if any).
+func (c *E3dbIdentityClient) UpdateRealmRole(ctx context.Context, params UpdateRealmRoleRequest) (*Role, error) {
+	var role *Role
+	path := c.Host + identityServiceBasePath + "/" + realmResourceName + "/" + params.RealmName + "/" + roleResourceName + "/" + params.RoleID
+	req, err := e3dbClients.CreateRequest("DELETE", path, params.Role)
```

Comment:
Should this be a "PUT"?

Reply:
This is why my tests keep saying bad request!